tests/specs/functional/lemmas-spec.k

```diff
@@ -122,3 +123,10 @@ module LEMMAS-SPEC
            => doneLemma( true ) ... </k>
 
 endmodule
+
+module LEMMAS-JAVA-SPEC [symbolic, kast]
+    imports VERIFICATION
+
+    claim <k> runLemma ( #bufStrict( #ceil32(LEN) -Int LEN, 0 ) ) => doneLemma( #buf( #ceil32(LEN) -Int LEN, 0 ) ) ... </k>
+
+endmodule
```

Comment:
Why is this only run with the Java backend? Also, are we sure that this actually even gets run as a lemma? I've never seen claims imported into the main verification module.

Reply:
I was not sure of this as well, as I explaind in the description. Fixed by adding a new proof file `lemmas-java-spec.k`.

It is only run on Java because lemmas were only added on Java, for performance.